mne/viz/_brain/_brain.py

```diff
@@ -2496,6 +2536,157 @@ def add_foci(self, coords, coords_as_verts=False, map_surface=None,
                                   opacity=alpha, resolution=resolution)
             self._renderer.set_camera(**views_dicts[hemi][v])
 
+    @verbose
+    def add_sensors(self, inst, trans, picks=None, colors=None,
+                    scales=None, alphas=None, meg_helmet=True,
+                    project_eeg=False, fnirs_pairs=None, fnirs_sources=None,
+                    fnirs_detectors=None, verbose=None):
+        """Add mesh objects to represent sensor positions.
+
+        Parameters
+        ----------
+        inst : mne.Info | mne.io.Raw | mne.Epochs | mne.Evoked
```

Comment:
I would suggest you only accept an Info.

Reply:
This is @larsoner's request for forward compatibility when data may be added as well as channel positions

Reply:
Let's start with just `info` and add support for the others later, when we actually want to plot data values